worlds/shorthike/__init__.py

```diff
@@ -92,20 +107,23 @@ def create_regions(self) -> None:
             self.multiworld.completion_condition[self.player] = lambda state: state.has("Golden Feather", self.player, 12)
         elif self.options.goal == "races":
             # Races
-            self.multiworld.completion_condition[self.player] = lambda state: (state.has("Golden Feather", self.player, get_min_feathers(self, 7, 9))
-                or (state.has("Bucket", self.player) and state.has("Golden Feather", self.player, 7)))
+            self.multiworld.completion_condition[self.player] = lambda state: state.can_reach_location("Hawk Peak Race Reward", self.player)
         elif self.options.goal == "help_everyone":
             # Help Everyone
-            self.multiworld.completion_condition[self.player] = lambda state: (state.has("Golden Feather", self.player, 12)
-                and state.has("Toy Shovel", self.player) and state.has("Camping Permit", self.player)
-                and state.has("Motorboat Key", self.player) and state.has("Headband", self.player)
-                and state.has("Wristwatch", self.player) and state.has("Seashell", self.player, 15)
-                and state.has("Shell Necklace", self.player))
+            self.multiworld.completion_condition[self.player] = lambda state: (state.can_reach_location("Collect 15 Seashells", self.player)
+                and state.can_reach_location("Ranger May Shell Necklace Golden Feather", self.player)
+                and state.can_reach_location("Sue the Rabbit Shoes Reward", self.player)
+                and state.can_reach_location("Wristwatch Trade", self.player)
+                and state.can_reach_location("Tough Bird Salesman (400 Coins)", self.player)
+                and state.can_reach_location("Return Camping Permit", self.player)
+                and state.can_reach_location("Shovel Kid Trade", self.player)
+                and state.can_reach_location("Boat Challenge Reward", self.player)
+                and state.can_reach_location("Purchase Sunhat", self.player)
+                and state.can_reach_location("Artist Golden Feather", self.player)
+                and state.has("Golden Feather", self.player, 12))
```

Comment:
Super small & not import optimisation comment if you want to, feel free to just ignore

Order these from "least likely to be true" to "most likely to be true"
(Unless you already did that, in which case, nice :D)

*low priority*